gcn/circulars.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/circulars.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "GCN Circular",
+  "description": "GCN Circulars are rapid astronomical bulletins submitted by and distributed to community members worldwide. They are used to share discoveries, observations, quantitative near-term predictions, requests for follow-up observations, or future observing plans related to high-energy, multi-messenger, and variable or transient astrophysical events. See https://gcn.nasa.gov/circulars",
+  "properties": {
+    "eventId": {
+      "type": "string",
+      "description": "Event name, automatically inferred from the subject"
+    },
+    "submitter": {
```

Comment:
Shall we make it array of [name, affiliation, email]?

Reply:
No, because this field is not guaranteed to be machine-readable and parseable in that way.